examples/agent-with-priority-class.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: agent-as-daemonset
+spec:
+  agent:
+    strategy: DaemonSet
+    PriorityClassName: test # priorityClassName here
```

```suggestion
    priorityClassName: test # priorityClassName here
```

This example should include an example of a priority class, like (from the documentation):

```yaml
apiVersion: scheduling.k8s.io/v1
kind: PriorityClass
metadata:
  name: high-priority
value: 1000000
globalDefault: false
description: "This priority class should be used for XYZ service pods only."
```

If you use this in the example, don't forget to change the name to `high-priority` in the Jaeger CR.

---

yes, sorry.
added changes as above
@jpkrohling pls review